modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1269,6 +1273,23 @@ public void testAddObjectEntryWithBooleanObjectField() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddObjectEntryWithDefaultLanguageId() throws Exception {
+		_testAddObjectEntryWithDefaultLanguageId(
+			ObjectDefinitionTestUtil.publishObjectDefinition());
+		_testAddObjectEntryWithDefaultLanguageId(
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Collections.singletonList(
+					new TextObjectFieldBuilder(
+					).labelMap(
+						LocalizedMapUtil.getLocalizedMap(
+							RandomTestUtil.randomString())
+					).name(
+						"a" + RandomTestUtil.randomString()
+					).build()),
+				ObjectDefinitionConstants.SCOPE_SITE));
```

Comment:
Can you check if these object definitions are being deleted after running this test?

Reply:
I added a call do delete them at the end